testutil/testutil.go

```diff
@@ -124,24 +124,16 @@ func createExclusionsList(exclusionsText string, t *testing.T) []portpair {
 	return exclusions
 }
 
-// LimitedWriter is an io.Writer that will return an EOF error after MaxLen has
-// been reached.  If MaxLen is 0, Writes will always succeed.
-type LimitedWriter struct {
-	bytes.Buffer
-	MaxLen int
+// ErrorWriter is an io.Writer that will return an error all ways
+type ErrorWriter struct {
```

Comment:
This should be internal in the file_exporter_test.go  file.

Reply:
Resolved